packages/account-wasm/src/types/policy.rs

```diff
@@ -25,19 +25,21 @@ impl TryFrom<JsValue> for Policy {
 impl TryFrom<Policy> for account_sdk::account::session::hash::Policy {
     type Error = EncodingError;
 
-    fn try_from(value: Policy) -> Result<Self, Self::Error> {
-        Ok(Self {
-            contract_address: Felt::from_str(&value.target)?,
-            selector: get_selector_from_name(&value.method).unwrap(),
-        })
+    fn try_from(_value: Policy) -> Result<Self, Self::Error> {
+        todo!();
+        // Ok(Self {
```

Comment:
Should I modify this accordingly, or will the other guys do it to their liking?

Reply:
Yes let's update these types